web/ci.sh

```diff
@@ -83,4 +90,121 @@ if builder_start_action validate-size; then
   ./src/tools/building/check-build-size.sh $FLAGS
 
   builder_finish_action success validate-size
+fi
+
+if builder_start_action publish-s.keyman; then
+  # First phase: make sure the s.keyman.com repo is locally-available and up to date.
+  pushd "$S_KEYMAN_COM"
+  if builder_has_option --password; then
+    git pull https://keyman-server:$PASSWORD@github.com/keymanapp/s.keyman.com.git master
+  else
```

Comment:
Note that the on-server script must specify the password; this script does not reference the actual environment variable / parameter name.